CallAutomation_SimpleIvr/Program.cs

```diff
@@ -126,21 +113,30 @@ await client.GetCallConnection(@event.CallConnectionId)
             }
             else
             {
-                PlaySource invalidAudio = new FileSource(new Uri(audioBaseUrl + builder.Configuration["InvalidAudio"]));
+                PlaySource invalidAudio = new FileSource(new Uri(baseUri + builder.Configuration["InvalidAudio"]));
                 await client.GetCallConnection(@event.CallConnectionId).GetCallMedia().PlayToAllAsync(invalidAudio, audioPlayOptions);
             }
         }
         if (@event is RecognizeFailed { OperationContext: "MainMenu" })
         {
 
             // play invalid audio
-            await client.GetCallConnection(@event.CallConnectionId).GetCallMedia().PlayToAllAsync(new FileSource(new Uri(audioBaseUrl + builder.Configuration["InvalidAudio"])), new PlayOptions() { Loop = false });
-            await client.GetCallConnection(@event.CallConnectionId).HangUpAsync(true);
+            await client.GetCallConnection(@event.CallConnectionId).GetCallMedia().PlayToAllAsync(new FileSource(new Uri(baseUri + builder.Configuration["InvalidAudio"])), audioPlayOptions);
         }
         if (@event is PlayCompleted { OperationContext: "SimpleIVR" })
         {
             await client.GetCallConnection(@event.CallConnectionId).HangUpAsync(true);
         }
+        if(@event is PlayCompleted { OperationContext: "AgentConnect" })
```

Comment:
We should also handle PlayFailed event for "AgentConnect" operation context. May be disconnect the call and add logging as well

Reply:
since we required to add agent, so hangup will drop the call, however updated both events into single if now.